cdap-common/bin/service

```diff
@@ -208,6 +210,7 @@ run() {
   # Setup classpaths.
   cdap_set_classpath "${COMPONENT_HOME}" "${CDAP_CONF}"
   cdap_set_hive_classpath || exit 1
+  cdap_set_spark
```

Comment:
Yes, `start_java` isn't optional. I'm not aware that Spark would ever be necessary here, but I have added it for silent support, if present. So far, it's only been necessary to include things that create persistent storage/Datasets.